templates/react/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/frontend/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "my_localstack_extension",
+  "version": "0.1.0",
+  "private": true,
+  "license": "UNLICENSED",
+  "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@localstack/theme": "git+ssh://git@github.com/localstack/localstack-web.git#commit=741276a31896cb498dc6207830842e6cbcca9611&workspace=@localstack/theme",
```

Comment:
see my comment on slack, would be great to clarify how to deal with closed/open source friction